tests/e2e/concurrent-e2e/concurrent_test.go

```diff
@@ -348,7 +348,7 @@ var _ = Describe("Concurrent e2e", Serial, func() {
 				rollout.Spec.Strategy = &apiv1.PipelineTypeRolloutStrategy{
 					PipelineTypeProgressiveStrategy: apiv1.PipelineTypeProgressiveStrategy{
 						Progressive: apiv1.ProgressiveStrategy{
-							AssessmentSchedule: "240,60,10",
+							AssessmentSchedule: "240,600,60,10",
```

Comment:
Did these really need to be set to 600 in order for it to pass, or were you just being extra conservative?